adapter_pipelines/Optimus/adapter.wdl

```diff
@@ -0,0 +1,103 @@
+import "Optimus.wdl" as Optimus
+import "submit.wdl" as submit_wdl
+
+
+task GetInputs {
+  String bundle_uuid
+  String bundle_version
+  String dss_url
+  Int retry_seconds
+  Int timeout_seconds
+
+  command <<<
+    python <<CODE
+    from pipeline_tools import utils
+
+    # Get bundle manifest
+    uuid = '${bundle_uuid}'
+    version = '${bundle_version}'
+    dss_url = '${dss_url}'
+    retry_seconds = ${retry_seconds}
+    timeout_seconds = ${timeout_seconds}
+    print('Getting bundle manifest for id {0}, version {1}'.format(uuid, version))
+    manifest_files = utils.get_manifest_files(uuid, version, dss_url, timeout_seconds, retry_seconds)
+
+    print('Downloading assay.json')
+    assay_json_uuid = manifest_files['name_to_meta']['assay.json']['uuid']
+    assay_json = utils.get_file_by_uuid(assay_json_uuid, dss_url)
+
+    # Parse inputs from assay_json and write to fastq_inputs
+    print('Writing fastq inputs to fastq_inputs.tsv')
+    lanes = assay_json['seq']['lanes']
```

Comment:
In v4.5.0 metadata schema, this has been changed to `assay_json[content]['seq']['lanes']`